charmcraft/env.py

```diff
@@ -21,10 +21,30 @@
 import os
 import pathlib
 import sys
+from typing import Optional
 
 from charmcraft.cmdbase import CommandError
 
 
+def get_managed_environment_snap_channel_default() -> str:
+    """Default channel to use when installing Charmcraft snap from Snap Store.
+
+    May someday be updated to use a track known to be compatible with the running
+    version.  For now, we default to latest/stable.
+
+    :returns: Channel string.
+    """
+    return "latest/stable"
```

Comment:
Ensure this is default track ready

```suggestion
    return "stable"
```

Reply:
Good point!

Reply:
Done